articles/install-guide/standalone.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Execute Q# programs without a driver in a host language 
+author: KittyYeungQ
+ms.author: kitty
+ms.date: 4/24/2020
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.standalone
+---
+
+# Q# Command Line Applications```
+
+Q# programs can be executed on their own, without a driver in a host language like C#, F#, or Python.
+
+1. Pre-requisites
+
+- [.NET Core SDK 3.1 or later](https://www.microsoft.com/net/download)
+
+2. Installation
+
+While you can build Q# command line applications in any IDE, we highly recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. By using VS Code or Visual Studio and the QDK Visual Studio Code extension you gain access to richer functionality.
+These are also convenient tools for using [Q# and Python together](xref:todo) or for using [Q# and C# together](xref:todo), if you choose to use those workflows instead.
```

Comment:
Do we need this line? If the reader is here I don't think they need to know this. I think those todo are part of the warning as well.

Reply:
I was referring to the text starting at line 22

Reply:
Got it. We can remove the Python and C# mentions.